app/models/broadcast_event/plate_cherrypicked.rb

```diff
@@ -32,6 +32,13 @@ def initialize(args)
     _add_destination_plate
   end
 
+  def user_identifier
+    # allows for user identifiers that aren't in the SS db
+    return properties[:user_identifier] if properties[:user_identifier]
+
+    super
+  end
```

Comment:
Actually, thinking about it, this may be better as:

properties[:user_identifier].presence || super

As that would fallback to UNKNOWN if user_identifier was ''

Reply:
https://guides.rubyonrails.org/active_support_core_extensions.html#presence

Reply:
Good spot, have changed as per your suggestion.